classes/Options_V2.php

```diff
@@ -123,6 +123,20 @@ private function get( $key = null, $default = false ) {
 		return $default;
 	}
 
+	/**
+	 * Get only list of options.
+	 *
+	 * @since 3.0.0
+	 *
+	 * @param array $only only options.
+	 *
+	 * @return array
+	 */
+	public static function get_only( $only = array() ) {
```

Comment:
`$only` argument should accept string as well for a single option

Reply:
This is dedicated for fetching list of selected options. Mentioned in doc comment
``
Get only list of options.
``

We have already `get` and `tutor_utils()->get_option` for single option.